uart/lcd_uart/README.adoc

```diff
@@ -0,0 +1,44 @@
+= Attaching a 16x2 LCD via TTL
+
+This example code shows how to interface the Raspberry Pi Pico to one of the very common 16x2 LCD character displays. Due to the large number of pins these displays use, they are commonly used with extra drivers or backpacks. In this example, we will use an Adafruit LCD display backpack, which supports communication over USB or TTL. An RGB display is also used, but the backpack is compatible with white/blue or one color displays too.
+
+The backpack processes a set of commands that are documented https://learn.adafruit.com/usb-plus-serial-backpack/command-reference[here] and preceded by the "special" byte 0xFE. The backpack does the ASCII character conversion and even supports custom character creation. In this example, we use the primary UART (uart0) to read characters and print them onto the LCD. We also define a special startup sequence and vary the display's backlight color.
+
+[NOTE]
+======
+These LCD displays can also be driven directly using GPIO without the use of an adapter board. That is beyond the scope of this example and not very commonly done.
```

Comment:
"not very commonly done." - oh?

Reply:
While true — you don't often see people using these 16x2 displays these days without an I2C or TTL adaptor — this isn't really that relevant. People will either already know it, or if they don't then we shouldn't distract them with questions we're not going to answer. Just delete the note.

Reply:
I actually brought this one over from the I2C LCD example but makes sense.